python/paddle/v2/dataset/imdb.py

```diff
@@ -12,9 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-IMDB dataset: http://ai.stanford.edu/%7Eamaas/data/sentiment/aclImdb_v1.tar.gz
+IMDB dataset.
 
-TODO(yuyang18): Complete comments.
+This module download IMDB dataset from
+http://ai.stanford.edu/%7Eamaas/data/sentiment/, which contains a set of 25,000
```

Comment:
`, which contains a set of  25,000` ==> `. This dataset contains 25,000 reviews for training, and 25,000 for testing.`